recipes/DragAndDropHalogenHooks/src/Main.purs

```diff
@@ -69,40 +71,53 @@ hookComponent = Hooks.component \_ _ -> Hooks.do
           mbFileList = DataTransfer.files $ dataTransfer e
           fileArray = maybe [] FileList.items mbFileList
         Hooks.put filesIdx fileArray
+        Hooks.put hoverIdx false
     ]
-    -- Note: Elm uses a button that, when clicked, will do the following:
-    -- 1. create an input element
-    -- 2. add it to the DOM
-    -- 3. create a mouse event
-    -- 4. dispatch the mouse event to the input element
-    -- 5. (implication) file dialogue appears
-    -- 6. user selects a file
-    -- 7. input event handler runs a callback using user's selected file
-    -- 8. input element is removed from DOM
-    --
-    -- The approach used below is based on this SO answer:
-    -- https://stackoverflow.com/a/47094148
```

Comment:
If you're going to delete this, maybe it should be moved to the recipe's Readme?

Reply:
I just moved this section to the bottom of the file

Reply:
Ah, I hadn't scrolled that far... Thanks for clarifyin!